### ISSUE TITLE
branch: buildANucleusDeprecated

### ISSUE THREAD

**Issue by chrisklus** [phetsims/build-an-atom#226]
This branch will be used to make commits for https://github.com/phetsims/build-a-nucleus/issues/2 before we decide how they should be merged into master. The branch will be deleted once it is merged.

**Comment by Luisav1** [comment]
The development style of making build-a-nucleus run off of build-an-atom code (https://github.com/phetsims/build-a-nucleus/issues/2) has been abandoned. This branch will now be deleted once development for the new Build a Nucleus sim is farther along.

**Comment by zepumph** [comment]
I believe we are likely ready to delete this branch. @Luisav1 is that true?

**Comment by Luisav1** [comment]
Yes @zepumph, this branch is now deleted. Closing.
